omaha/protocol/protocol.go

```diff
@@ -26,6 +26,12 @@ type Protocol interface {
 	FormatWebStoreResponse(extension.Extensions, string) ([]byte, error)
 }
 
+type requestProtocol struct {
```

Comment:
Since the struct is used exclusively inside `DetectProtocolVersion` I'd move it there (it doesn't have to be global).

Reply:
https://github.com/brave/go-update/pull/276/commits/041f9296569bf218b13d8fcd06c2b529dc1f8211